content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,360 @@
+~~~~~~~~~~~~~~~~~
+
+The :doc:`chart of accounts <../accounting/get_started/chart_of_accounts>` is installed by default
+as part of the set of data included in the localization module, the accounts are mapped
+automatically in taxes, default accounts payable, and default accounts receivable.
+
+Accounts can be added or deleted according to the company's needs.
+
+.. seealso::
+   :doc:`../accounting/get_started/chart_of_accounts`
+
+Contacts
+~~~~~~~~
+
+To create a contact, navigate to :menuselection:`Contacts app` and select :guilabel:`New`. Then
+enter the following information:
+
+- :guilabel:`Company Name`
+- :guilabel:`Address`:
+
+  - :guilabel:`Street`: required to confirm an electronic invoice.
+  - :guilabel:`City`
+  - :guilabel:`State`
+  - :guilabel:`ZIP`
+  - :guilabel:`Country`: required to confirm an electronic invoice.
+
+- :guilabel:`Identification Number`:
+
+  - :guilabel:`Type`: select a identification type.
+  - :guilabel:`Number`: required to confirm an electronic invoice.
+
+Review taxes
+~~~~~~~~~~~~
+
+As part of the Uruguay localization module, taxes are automatically created with its configuration
+and related financial accounts.
+
+.. image:: uruguay/taxes.png
+   :alt: Taxes for Uruguay.
+
+Review document types
+~~~~~~~~~~~~~~~~~~~~~
+
+Some accounting transactions like *customer invoices* and *vendor bills* are classified by document
+types. These are defined by the government fiscal authorities, in this case by the |DGI|.
+
+Each document type can have a unique sequence per journal where it is assigned. The data is created
+automatically when the localization module is installed, and the information required for the
+document types is included by default.
+
+To review the document types included in the localization, navigate to :menuselection:`Accounting
+--> Configurations --> Document Types`.
+
+.. note::
+   In Uruguay, |CAEs| **must** be uploaded in Uruware. Sequences (and PDFs) are received in Odoo
+   from Uruware, based on their |CAEs|. |CAEs| are **only** used in production. When testing, only a
+   range of sequences used in Uruware need to be set.
+
+.. image:: uruguay/document-types.png
+   :alt: Document types for Uruguay.
+
+Sales journals
+~~~~~~~~~~~~~~
+
+To generate and confirm an electronic document that will be validated by |DGI|, the sales journal
+needs to be configured with the following:
+
+- :guilabel:`Invoicing Type`: manual invoices are for open invoices previously stamped in another
+  system, for example directly in the |DGI|. Electronic invoices will not be sent to Uruware.
+- :guilabel:`Use Documents?`: Activate this option if this journal will use documents from the list
+  of document types in Odoo.
+
+Workflows
+=========
+
+Once you have configured your database, you can create your documents.
+
+Sales documents
+---------------
+
+Customer invoices
+~~~~~~~~~~~~~~~~~
+
+:guilabel:`Customer invoices` are electronic documents that, when validated, are sent to |DGI| via
```

Comment:
```suggestion
Customer invoices are electronic documents that, when validated, are sent to |DGI| via
```
I would not use guilabel here since we're talking about the general term/concept. I would, however, add a reference to the relevant doc.